bundles/ranvier-combat/behaviors/item/decay.js

```diff
@@ -37,8 +37,12 @@ module.exports = srcPath => {
         }
 
         if (belongsTo) {
+          const owner = this.findOwner();
+          if (owner) {
+            Broadcast.sayAt(owner, `Your ${this.name} has rotted away!`);
+            Broadcast.prompt(owner);
```

Comment:
Ah, gotcha. It looked weird to me during testing to have it say that and then not prompt me. But I can remove it.